build.gradle

```diff
@@ -4,7 +4,7 @@ allprojects {
   apply plugin: 'java'
   apply plugin: 'checkstyle'
 
-  sourceCompatibility = 1.7
+  sourceCompatibility = 1.8
```

Comment:
I think probably should still keep sourceCompatibility = 1.7 until Apache Kafka has changed sourceCompatibility to 1.8.